### DISCUSSION TITLE
Admin setting are lost after every update

### DISCUSSION THREAD

**Discussion by mgielissen** [open-webui/open-webui#5175] [General]
Everytime when I update open-webui (tried with docker and pip3) admin settings are reset to default (like Open AI url and key, allow new users = true).

**Comment by sanebg** [comment]
I'm having the same exact issue, did you find a resolution?

**Comment by silentoplayz** [comment]
If anything was followed from the actual documentation, you'd have a persistent volume created for Open WebUI. https://docs.openwebui.com/getting-started/

**Reply by Mike-7777777** [reply]
Yes, but `pip install -U open-webui`? It will reset everything too, we have a solotion for that?

**Reply by Mike-7777777** [reply]
> > Yes, but `pip install -U open-webui`? It will reset everything too, we have a solotion for that?
> 
> That's a command to uninstall Open WebUI. I'd usually expect my settings to be reset for an application that I uninstall from my PC.

perfect
